recipe/install_ld.sh

```diff
@@ -9,5 +9,7 @@ make DESTDIR=$PWD/install install-strip
 CHOST="${ctng_cpu_arch}-conda-linux-gnu"
 OLD_CHOST="${ctng_cpu_arch}-${ctng_vendor}-linux-gnu"
 mkdir -p $PREFIX/bin
+mkdir -p $PREFIX/$OLD_CHOST/bin
 cp $PWD/install/$PREFIX/bin/$CHOST-ld $PREFIX/bin/$CHOST-ld
 ln -s $PREFIX/bin/$CHOST-ld $PREFIX/bin/$OLD_CHOST-ld
+ln -s $PREFIX/bin/$CHOST-ld $PREFIX/$OLD_CHOST/bin/ld
```

Comment:
Do we need to put `ld` in `$PREFIX/$CHOST/bin` too @isuruf?

Reply:
yes.

Reply:
done